src/jit/codegencommon.cpp

```diff
@@ -3981,10 +4001,12 @@ void            CodeGen::genFnPrologCalleeRegArgs(regNumber xtraReg,
         {
             // A struct might be passed  partially in XMM register for System V calls.
             // So a single arg might use both register files.
+#ifndef ARM_SOFTFP
             if (isFloatRegType(regType) != doingFloat)
```

Comment:
In fact I got it in the following way: if we just make
#define MAX_FLOAT_REG_ARG        0
then in the function prolog we get code just trying to read them from the stack.

And next if we don't make the changes in src/jit/register_arg_convention.h etc. to make it count the float arguments in the int registers we won't get them read from int registers at all.